src/test/java/hw21.java

```diff
@@ -0,0 +1,36 @@
+import org.example.pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Instant;
+
+import static org.example.WaitUtils.waitUntilVisibilityOfElementLocatedBy;
+import static org.openqa.selenium.By.*;
+public class hw21 extends BaseTest {
```

Comment:
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

public class **HomeWork21** extends BaseTest

Rename refactorings https://www.jetbrains.com/help/idea/rename-refactorings.html#rename_class_example